OpenMTRDemo/Filters/BaseFilter.cs

```diff
@@ -12,6 +12,7 @@ public partial class BaseFilter : UserControl
         public string FilterName;
         public ExpandedImageForm Editor;
         public FlowLayoutPanel FiltersPanel;
+        public string ListableName { get { return FilterName;  } }
```

Comment:
Unless I'm missing something here, can we use the getter with FilterName and do the exact same thing without an additional variable?

Reply:
Nope, you can't just point it to the variable directly, it has to be a property.

Reply:
I see.